server/helpers/loadConfig.js

```diff
@@ -0,0 +1,98 @@
+// config intended to be passed on to frontend
+// keys in process .env must be referencesd explicitly
+// so dotenv-webpack is able to include them in the test bundle for Mocha tests
+const check = require('check-types')
+const isBoolString = require('./isBoolStr')
+
+// process.env.npm_package_version is undefined on Heroku
+const appVersion = require('../../package.json').version
+
+const {
+  ADALITE_ENABLE_DEBUGGING,
+  ADALITE_SERVER_URL,
+  ADALITE_BLOCKCHAIN_EXPLORER_URL,
+  ADALITE_DEFAULT_ADDRESS_COUNT,
+  ADALITE_GAP_LIMIT,
+  ADALITE_DEMO_WALLET_MNEMONIC,
+  ADALITE_ENABLE_TREZOR,
+  ADALITE_LOGOUT_AFTER,
+  ADALITE_ENABLE_SERVER_MOCKING_MODE,
+  ADALITE_MOCK_TX_SUBMISSION_SUCCESS,
+  ADALITE_MOCK_TX_SUMMARY_SUCCESS,
+  TREZOR_CONNECT_URL,
+  REDIS_URL,
+  STATS_PWD,
+  PORT,
+} = process.env
+
+const checkMap = check.map(
+  {
```

Comment:
I would put there directly process.env and where integers need to be validated I would do something like this:
```
   ADALITE_DEFAULT_ADDRESS_COUNT: (x) => check.positive(parseInt(x, 10))
```
This should vastly eliminate the duplicity